Goal: Fix a memory leak in 4.10
Fixes: #301378

Status wrt upstream: Applied in CVS

Index: geneweb-4.10/src/secure.ml
===================================================================
--- geneweb-4.10.orig/src/secure.ml	2005-08-30 07:57:10.000000000 +0200
+++ geneweb-4.10/src/secure.ml	2005-08-30 08:14:18.000000000 +0200
@@ -13,7 +13,7 @@
 value decompose =
   loop [] where rec loop r s =
     let b = Filename.basename s in
-    if b = "" then
+    if b = "" || b = Filename.current_dir_name then
       let d = Filename.dirname s in
       if d = "" then r
       else if d = s then [d :: r]
